rwatch/graphics/graphics.c

```diff
@@ -90,16 +91,17 @@ GBitmap *graphics_capture_frame_buffer(n_GContext *context)
     return (GBitmap *)display_get_buffer();
 }
 
-GBitmap *graphics_capture_frame_buffer_format(n_GContext *context, GBitmap format)
+GBitmap *graphics_capture_frame_buffer_format(n_GContext *context, GBitmapFormat format)
 {
     // TODO Honestly not entirely sure what is expected here
     // rbl_lock_frame_buffer
     return (GBitmap *)display_get_buffer();
 }
 
-void graphics_release_frame_buffer(n_GContext *context, GBitmap *bitmap)
+bool graphics_release_frame_buffer(n_GContext *context, GBitmap *bitmap)
 {
     // rbl_unlock_frame_buffer
+    return false; //XXX
```

Comment:
XXX ? Add issue for this "Add lock and unlock framebuffer support"

Reply:
Created issue pebble-dev/RebbleOS/issues/38